### ISSUE TITLE
Teams custom app display is all grey when switched out and back to Teams app

### ISSUE THREAD

**Issue by ceroles** [OfficeDev/microsoft-teams-library-js#1844]
We have a Teams custom app and we have noticed a strange issue recently. When custom app is open, then put Teams in the background, then switch back to Teams, the display is just grey. This is also occurring with other custom apps and not just ours. Screenshot for reference: 
![Untitled](https://github.com/OfficeDev/microsoft-teams-library-js/assets/118153014/d529a231-6826-460d-aaa6-66fa81a5f2b5)

Replicated on iPad iOS version 16.5.1, Teams versions 5.12.1 and 5.12.0

Thank you.

**Comment by Prasad-MSFT** [comment]
@ceroles - Is the issue only specific to iPad? Are you also facing same issue in MS Teams Desktop/Web Client in Windows?
Are you facing same issue with 1st party app as well?

**Comment by ceroles** [comment]
Yes, we have so far only been able to replicate on iPad Teams app. Yes, we are actually able to replicate as well on Approvals, here is a video:
https://github.com/OfficeDev/microsoft-teams-library-js/assets/118153014/56fc1546-2661-41f9-9eec-44e80602d093

**Comment by Prasad-MSFT** [comment]
@ceroles - We have checked the 'Approval' app scenario in Windows machine at our end and observed that it works perfectly fine in both Teams Desktop `(Microsoft Teams Version 1.6.00.18871)`/Web client as well as in mobile devices.

We will check this behaviour in iPad and will update you.

**Comment by Prasad-MSFT** [comment]
Hi @ceroles - Could you please let us know if you are still able to repro this issue or not?
